RunLog/RunLog/Sources/Presentation/DetailLog/DetailLogView.swift

```diff
@@ -175,10 +177,19 @@ final class DetailLogView: UIView {
         stack.axis = .horizontal
         stack.alignment = .center
         stack.spacing = 16
-        stack.distribution = .fillEqually
+        stack.distribution = .equalSpacing
         return stack
     }()
     
+    let recordTitleLabel = UILabel().then {
+        $0.text = "기록 상세"
+        $0.font = .RLHeading1
+        $0.textColor = .Gray000
```

Comment:
```suggestion
$0.attributedText = .RLAttributedString(text: "기록 상세", font: .Heading1)
```
font에 RL쓰는 부분은 버튼 타이틀이 아닌 이상 attributedString을 쓰신다고 보면 됩니다